components/list/list-item-generic-layout.js

```diff
@@ -56,18 +57,45 @@ class ListItemGenericLayout extends LitElement {
 		`;
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+
+		this._preventFocusFromNonActions();
+	}
+
 	render() {
 		return html`
+		<slot name="content-action"></slot>
+		<slot name="outside-control-action"></slot>
+		<slot name="control-action"></slot>
+		<slot name="actions"></slot>
+
 		<slot name="outside-control"></slot>
 		<slot name="control"></slot>
 		<slot name="content"></slot>
-		<slot name="actions"></slot>
-
-		<slot name="outside-control-action"></slot>
-		<slot name="control-action"></slot>
-		<slot name="content-action"></slot>
 		`;
 	}
+
+	_preventFocusFromNonActions() {
+		const slots = [
+			'content',
+			'control',
+			'outside-control'
+		];
+		for (const slot of slots) {
+			const content = this.querySelector(`[slot="${slot}"]`);
+			const focusables = content ? getAllFocusableDescendants(content) : [];
+			for (const focusable of focusables) {
+				// remove focus and click events from focusable items.
+				// Items requiring focus MUST be placed within an action area
+				focusable.setAttribute('tabindex', '-1');
```

Comment:
This was the cleanest way I could figure to prevent focus reliably - alternatives if this is not ideal are welcome